frontend/src/views/Pages/UserProfile/UserCard_FormUsername.vue

```diff
@@ -47,16 +63,27 @@ export default {
   name: 'FormUsername',
   data() {
     return {
-      edit_username: true,
-      username: '',
+      editUsername: true,
+      username: this.$store.state.username,
+      form: {
+        username: this.$store.state.username,
+      },
     }
   },
+  props: {
+    UserProfileTestData: { type: Object },
+  },
   methods: {
     async onSubmit() {
-      // console.log(this.data)
-      const result = await loginAPI.changeUsernameProfile(this.username)
+      const result = await loginAPI.changeUsernameProfile(
+        this.$store.state.sessionId,
+        this.$store.state.email,
+        this.form.username,
+      )
       if (result.success) {
-        alert('changeUsername success')
+        this.$store.commit('username', this.form.username)
+        this.editUserdata = this.editUsername = !this.editUsername
+        alert('Dein Username wurde geändert.')
```

Comment:
```suggestion
        alert('Dein Username wurde geändert.')
```

Can we remove this  and show the user in a different form?